src/include/test/array_defs.h

```diff
@@ -302,10 +303,31 @@ static std::filesystem::path diskann_mem_index{
 static std::filesystem::path diskann_truth_index_data =
     diskann_root / "truth_index_siftsmall_learn_256pts_R4_L50_A1.2.data";
 
-static std::filesystem::path pytest_170_group_root{
-    test_array_root / "pytest-170"};
-static std::filesystem::path pytest_170_group_uri{
-    pytest_170_group_root / "test_ivf_flat_ingestion_f320/array"};
+using vamana_nano_feature_type = float;
+using vamana_nano_groundtruth_type = uint64_t;  // int32?
```

Comment:
Could you add more context on this comment?

Reply:
Added the following comment and @todo item:
```c++
/*                                                                                                                                                                                                                                                                                             
 * Definitions for IVF index for "sift" (10k subset on http://corpus-texmex.irisa.fr)                                                                                                                                                                                                          
 * Here, because of some hard-coded typing in the Python implementation, groundtruth                                                                                                                                                                                                           
 * is typed as uint64_t -- however, in the reference data from the website, the                                                                                                                                                                                                                
 * grountruth type is uint32_t.                                                                                                                                                                                                                                                                
 *                                                                                                                                                                                                                                                                                             
 * @todo Create groundtruth with uint32_t as well as ids_type and indices_type as                                                                                                                                                                                                              
 * uint32_t.  Verify they can be mixed and matched.                                                                                                                                                                                                                                            
 */
```

The comment was there for two reasons:
1) The original data has 32bit groundtruth
2) We should support multiple types, including uint32_t and be able to mix and match them